### PR TITLE
add survey purpose

### PR DESCRIPTION
**Pull request by steveoh**
refs #59

need to figure out how values get linked to drop down still

### CODE REVIEW

**Comment by steveoh** [comment]
this seems to be working

**Comment by steveoh** [comment]
image

File: src/app/location/Location.js [hunk 1]
```diff
@@ -246,7 +246,9 @@ define([
                     // this needs to be here since it sets the text on the verify btn
                     // and it needs to wait for $(that.verifyMapBtn).button('loading'); to finish
                     // which is async
-                    that.addLineToMap(inProgressData.path);
+                    if (inProgressData && inProgressData['path']) { // eslint-disable-line dot-notation
```
Comment (stdavis) [inline src/app/location/Location.js]: Same comment as in the other PR. No need to use brackets (and related eslint comment) here.

**Comment by stdavis** [comment]
Also, there's something strange with the travis build for "push". I tried restarting it but it seemed to never restart. Maybe it's not a huge deal.

File: src/app/location/Location.js (hunk 1 above)
Comment (steveoh) [reply src/app/location/Location.js]: switched to dot notation

**Comment by steveoh** [comment]
all the tests pass on my machine!

**Pull request merged**